src/cli/utils/sharepoint_test.go

```diff
@@ -180,13 +180,6 @@ func (suite *SharePointUtilsSuite) TestIncludeSharePointRestoreDataSelectors() {
 			},
 			expectIncludeLen: 2,
 		},
-		{
```

Comment:
@ryanfkeepers - why is this removed?

Reply:
It isn't actually covered within that test.  I plugged it in with the last PR, then forgot to take it out when I realized it was the wrong test.